action/action.go

```diff
@@ -0,0 +1,38 @@
+package action
+
+import "fmt"
+
```

Comment:
How about applying https://github.com/golang/go/wiki/CodeReviewComments to the code?

Specifically, around adding documentation to package exports.

Reply:
Comments added